website/src/content/concepts/oauth-scopes-and-user-permissions.mdx

```diff
@@ -42,7 +81,7 @@ Each Custom Application gets a **unique pair** of user permissions: "view" and "
 
 <Info>
 
-The permission names are unique to each Custom Application and they derive from the `entryPointUriPath`, based on the following format: `{View,Manage}<EntryPointUriPath>`.
+The permission names are unique to each Custom Application and, by default, they derive from the `entryPointUriPath`, based on the following format: `{View,Manage}<EntryPointUriPath>`.
```

Comment:
We have this sentence above:
> Each Custom Application gets a **unique pair** of user permissions: "view" and "manage."

I'm wondering if we should update that because a Custom Application using `additionalOAuthScopes` will not only get one pair of user permissions.